Makefile

```diff
@@ -100,10 +89,19 @@ SPHINX_BUILD_GOOGLE_DOCSTRINGS = sphinx-apidoc
 SPHINX_NAPOLEAN_BUILD_DIR = docs/source/gen
 SPHINX_PROJECT_DIR = tf_encrypted
 
+CONVERT_DIR=tf_encrypted/convert
+BUILD_RESERVED_SCOPES=$(CONVERT_DIR)/specops.yaml
+$(BUILD_RESERVED_SCOPES): pythoncheck
+	python -m tf_encrypted.convert.gen.generate_reserved_scopes
+
+BUILD_CONVERTER_README=$(CONVERT_DIR)/gen/readme_template.md
+$(BUILD_CONVERTER_README): $(BUILD_RESERVED_SCOPES) pythoncheck
+	python -m tf_encrypted.convert.gen.generate_reserved_scopes
+
 google-docstrings:
 	@$(SPHINX_BUILD_GOOGLE_DOCSTRINGS) -fMeET "$(SPHINX_PROJECT_DIR)" -o "$(SPHINX_NAPOLEAN_BUILD_DIR)"
 
-docs: google-docstrings
+docs: $(BUILD_CONVERTER_README) google-docstrings
```

Comment:
I know it's technically correct, but I don't really like the converter readme being built with the defunct Sphinx docs. this makes it really easy for people to forget to build it

Reply:
what if we let `build` depend on `docs`?

Reply:
that sounds good to me!

Reply:
Addressed in https://github.com/tf-encrypted/tf-encrypted/issues/779